README.md

```diff
@@ -5,13 +5,26 @@
 ## Drupal Starshot
 Starshot is Drupal 10, but supercharged with some of the best modules and themes out there, set up in useful ways to help you get started building a site right away. Starshot is built on the [Drupal recipe system](https://drupal.org/project/distributions_recipes) (soon to be in core), so it doesn't lock you in like a normal distribution would.
 
-## Installation
+### Installation
 ```
 composer create-project drupal/starshot-project --repository='{"type":"vcs","url":"https://github.com/phenaproxima/starshot-prototype"}' --stability=dev
 ```
 This one command will install Starshot and open it in a web browser for you to play with. You'll get all the modules and themes listed below, pre-configured.
 
-## Included modules and themes
+#### Which database?
+By default, Starshot sets up a SQLite database, so that you don't need to do any extra setup to get started.
+
+But if you want to install into another database, you can use the following commands instead of the one above, substituting the real database URL instead of `mysql://drupal:drupal@localhost/starshot`
+
+```
+composer create-project drupal/starshot-project --repository='{"type":"vcs","url":"https://github.com/phenaproxima/starshot-prototype"}' --stability=dev --no-install
```

Comment:
> the post-create-project-cmd script runs even if nothing has been installed.

@phenaproxima we can prevent that with `--no-scripts`. I tested the following, and it worked for me.

```suggestion
composer create-project drupal/starshot-project --repository='{"type":"vcs","url":"https://github.com/phenaproxima/starshot-prototype"}' --stability=dev --no-install --no-scripts
```